index.md

```diff
@@ -26,10 +26,10 @@ title: Distributed SQL query engine for big data
     <div class="col-1"></div>
     <div class="col-6" style="text-align: center;">
       <p>
-        <h1 style="color: #FFFFFF;">Trino Fest</h1>
-        <h3 style="color: #FFFFFF; padding-bottom: 1rem;">13 June, 2024</h3>
```

Comment:
do we want a subtitle and a shorter wording on the button instead @monimiller @colebow ?